tests/container-nesting

```diff
@@ -97,6 +121,9 @@ for poolDriver in ${poolDriverList}; do
 
     # Cleanup.
     lxc delete --force "${instName}"
+    if echo "${nestedPoolDriverList}" | grep -q zfs; then
```

Comment:
When matching in lists, it's best to use `-w` to avoid matching on substrings. Granted, the risk of having another driver with `zfs` in it's name is very low.
```suggestion
    if echo "${nestedPoolDriverList}" | grep -qwF zfs; then
```